src/azure-cli/setup.py

```diff
@@ -58,7 +58,6 @@
     'azure-cosmos~=3.0,>=3.0.2',
     'azure-data-tables==12.4.0',
     'azure-datalake-store~=0.0.53',
-    'azure-graphrbac~=0.60.0',
```

Comment:
According to https://dev.azure.com/azclitools/public/_build/results?buildId=186297&view=logs&jobId=61bf4700-6d0f-55d3-0cc1-f8d3dd51c404&j=61bf4700-6d0f-55d3-0cc1-f8d3dd51c404&t=395ca4f7-a79a-5a8d-bfb6-7c58df524136, the removal of `azure-graphrbac` SDK breaks 2 extensions:

- `aks-preview`
- `k8s-runtime`